examples/http/main.go

```diff
@@ -19,12 +19,20 @@ import (
 	"github.com/beatlabs/patron/log/std"
 )
 
+var assestsFoder string
+
 func init() {
 	err := os.Setenv("PATRON_JAEGER_SAMPLER_PARAM", "1.0")
 	if err != nil {
 		fmt.Printf("failed to set sampler env vars: %v", err)
 		os.Exit(1)
 	}
+	// allows to run from any folder the 'go run examples/http/main.go'
```

Comment:
I have introduced this environment variable so that the service http/main.go could be run from any folder and not just from the project root directory.